app/components/containers/ProducerMetrics.jsx

```diff
@@ -4,6 +4,8 @@ import Metric from '../Metric.jsx';
 // import DataSize from '../metrics/producer/DataSize.jsx';
 // import ProcessingTime from '../metrics/producer/ProcessingTime.jsx';
 
+// Electron App:
+// Renders Metric components for: DATA SIZE, PROCESSING TIME, PENDING DURATION
```

Comment:
you have the same comment for ConsumerMetrics.jsx and ProducerMetric.jsx, so it looks like they're doing the exact same thing. i'd make both of them a little more specific so it's clear why there are 2. (just mention that one is for a consumer and the other is for a producer)